Sources/StreamChatSwiftUI/ChatChannel/MessageList/Polls/PollAttachmentView.swift

```diff
@@ -181,16 +181,23 @@ struct SuggestOptionModifier: ViewModifier {
             content
                 .alert(title, isPresented: $showingAlert) {
                     TextField(L10n.Alert.TextField.pollsNewOption, text: $text)
-                    Button(L10n.Alert.Actions.cancel) {
+                    Button(L10n.Alert.Actions.cancel, role: .cancel) {
                         showingAlert = false
                     }
                     Button(L10n.Alert.Actions.add, action: submit)
                 } message: {
                     Text("")
                 }
         } else {
-            // TODO: Add for iOS < 15.
             content
+                .uiAlertController(
```

Comment:
When I add an option, the popup appears again

Reply:
Fixed! Thank you.